lib/fog/compute/google/models/instance_group.rb

```diff
@@ -27,6 +27,25 @@ def destroy(_async = true)
           data = service.delete_instance_group(name, zone_name)
         end
 
+        def add_instance(instance_name)
+          requires :identity, :zone
+
+          service.add_instance_group_instance(identity, zone_name, instance_name)
+        end
+
+        def remove_instance(instance_name)
+          requires :identity, :zone
+
+          service.remove_instance_group_instance(identity, zone_name, instance_name)
+        end
+
+        def list_instances
```

Comment:
This will return:
```
=> [{"instance"=>"https://www.googleapis.com/compute/v1/projects/dev-utility-90115/zones/europe-west1-d/instances/chef-server", "status"=>"RUNNING"},
 {"instance"=>"https://www.googleapis.com/compute/v1/projects/dev-utility-90115/zones/europe-west1-d/instances/elk-server", "status"=>"RUNNING"}]
```
@icco @plribeiro3000  - PTAL - is such a return acceptable or should I return an array of self-links?